spring-integration-mqtt/src/main/java/org/springframework/integration/mqtt/inbound/Mqttv5PahoMessageDrivenChannelAdapter.java

```diff
@@ -384,4 +436,28 @@ public void acknowledge() {
 
 	}
 
+	private class MessageListener implements IMqttMessageListener {
+
+		MessageListener() {
+		}
+
+		@Override
+		public void messageArrived(String topic, MqttMessage message) {
+			Mqttv5PahoMessageDrivenChannelAdapter.this.messageArrived(topic, message);
+		}
+
+	}
+
+	private class AdapterConnectCallback implements ClientManager.ConnectCallback {
```

Comment:
I think instead of this extra internal class we can just use a lambda to delegate to that `connectComplete()` method.